lnwallet/channel.go

```diff
@@ -97,6 +97,13 @@ var (
 	// both parties can retrieve their funds.
 	ErrCommitSyncRemoteDataLoss = fmt.Errorf("possible remote commitment " +
 		"state data loss")
+
+	// errCancelledAdd is returned when we have cancelled an htlc with
+	// a settle/fail, and the add no longer exists when we go to look it up.
+	// This is encountered when upgrading from earlier versions of LND where
+	// we would erroneously store the unsignedAckedUpdates longer than we
+	// had to.
+	errCancelledAdd = fmt.Errorf("cancelled add - lookup failed")
```

Comment:
So this'll end up causing a borked channel state (leading to a force close) for any user that happens to have their channel in this state at upgrade time?